1.0.0 changes

cs-abhishek-shukla opened the pull request:
- Added a brief overview
- Added installation and configuration as placeholders
- Wrote the usage section
- Contents pending

cs-rohit-khune commented on README.md:
```diff
@@ -1 +1,23 @@
+- **Version**:  1.0.0
+- **Certified**: No
+- **Publisher**: Fortinet 
+- **Compatible Version**: FortiSOAR v7.6.2 and later
```
This is FCP only solution pack, Min Compatibility should be FCP  25.2.b

cs-abhishek-shukla replied:
Done

cs-abhishek-shukla committed:
1.0.0 Changes:
- 1.0.0 changes
- Added a brief overview
- Added installation and configuration as placeholders
- Wrote the usage section

cs-rohit-khune commented on README.md:
```diff
@@ -1 +1,23 @@
+
+# Overview
+
+The **FortiRecon Automation Services** solution pack provides automated playbooks to streamline incident response for External Attack Surface Management (EASM) and Brand Protection (BP) use cases. It enables security analysts to efficiently manage *Critical* and *High-severity* issues by automating ticket creation in Jira and sending notifications for relevant services.
```
Include Adversary Centric Intelligence (ACI) in the Overview alongside EASM and BP.

cs-abhishek-shukla replied:
Done

cs-rohit-khune commented on README.md:
```diff
@@ -1 +1,23 @@
+
+The **FortiRecon Automation Services** solution pack provides automated playbooks to streamline incident response for External Attack Surface Management (EASM) and Brand Protection (BP) use cases. It enables security analysts to efficiently manage *Critical* and *High-severity* issues by automating ticket creation in Jira and sending notifications for relevant services.
+
+Key use cases include:
```
Modify this point in context of "Ticketing Automation" and "Intelligence Reporting" like mentioned in  usage.md

cs-abhishek-shukla replied:
Done

cs-abhishek-shukla committed:
Review comments, updated overview

cs-rohit-khune reviewed (approved):
LGTM
